x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -2005,7 +2005,7 @@ var security = (function () {
                 {from: "winlog.event_data.ProcessId", to: "process.pid", type: "long"},
                 {from: "winlog.event_data.ProcessName", to: "process.executable"},
                 {from: "winlog.event_data.IpAddress", to: "source.ip", type: "ip"},
-                {from: "winlog.event_data.ClientAddress", to: "related.ip", type: "ip"},
+                {from: "winlog.event_data.IpAddress", to: "related.ip", type: "ip"},
```

Comment:
Does the earlier line that renames `winlog.event_data.IpAddress` to `source.ip` cause a problem for this? I wonder if a more generic copy from `source.ip`/`destination.ip` to `related.ip` that gets applied to all events would be better (not sure if there's already something like this in the pipeline).

Reply:
Hi @andrewkroh, I agree with you. 
I'll populate the related.ip from the generic fields, I'll test it and commit the modifications